apps/vr-tests-react-components/src/stories/Positioning/Positioning.stories.tsx

```diff
@@ -843,6 +843,70 @@ const TargetDisplayNone = () => {
   );
 };
 
+const ShiftToCoverTargetWithAutoSize = () => {
+  const styles = useStyles();
+  const [overflowBoundary, setOverflowBoundary] = React.useState<HTMLDivElement | null>(null);
+  const { containerRef, targetRef } = usePositioning({
+    position: 'below',
```

Comment:
Let's add a test for horizontal axis too

Reply:
Added test:
<img width="334" alt="Screenshot 2024-12-16 at 16 21 31" src="https://github.com/user-attachments/assets/21c5e54e-0068-42af-8939-9356bd6caed0" /> 
=> 
<img width="333" alt="Screenshot 2024-12-16 at 16 21 35" src="https://github.com/user-attachments/assets/79c4d391-7d37-465f-a535-7638172e07d3" />